hlib/python/hlib/frontend/tests/test_batch_norm.py

```diff
@@ -0,0 +1,39 @@
+import heterocl as hcl
```

Comment:
Under the ``tests`` folder, there is a folder ``saved_models``. Should it be moved to ``samples``?

Reply:
Oh yeah